### DISCUSSION TITLE
Support and custom development?

### DISCUSSION THREAD

**Discussion by magick93** [adrien2p/medusa-extender#73] [Q&A] [answered]
Hello @adrien2p 

Are you available for contract work?

I'd be interested in work with you on a medusa-extender based project.

**Comment by adrien2p** [comment]
Hi @magick93 👋 it is something we can discuss, to be honest it really depends on the time investment needed since I am quite busy. Could you tell me more about your needs. 

Also, depending on what you really need, we could see if some sponsoring for consultancy with a certain amount of hours per month could be something doable for you 👌

You can email me directly to talk a bit more or you can also contact me on discord 🤘

**Comment by adrien2p** [comment, marked as answer]
Hey man 🤘
I probably got someone who could help you out, let me know if you are interested. You can send me an email
